backend/src/main/java/gov/cdc/usds/simplereport/config/SecurityConfiguration.java

```diff
@@ -37,9 +37,9 @@
     private static final Logger LOG = LoggerFactory.getLogger(SecurityConfiguration.class);
 
     public interface OktaAttributes {
-        public static final String EMAIL = "email";
-        public static final String FIRST_NAME = "given_name";
-        public static final String LAST_NAME = "family_name";
+        String EMAIL = "email";
```

Comment:
I would prefer to discuss why I do this than just get into edit wars where we make stylistic changes to each other's code all the time. 😁